examples/github/remix.config.js

```diff
@@ -1,10 +1,10 @@
 /**
  * @type {import('@remix-run/dev/config').AppConfig}
  */
-module.exports = {
+ module.exports = {
   appDirectory: "app",
   browserBuildDirectory: "public/build",
   publicPath: "/build/",
-  serverBuildDirectory: "server/build",
+  serverBuildDirectory: "build",
```

Comment:
Why change this? I prefer to keep the server build inside the server directly, it should work correctly.